buildifier/buildifier.go

```diff
@@ -39,6 +40,7 @@ var (
 	dflag = flag.Bool("d", false, "alias for -mode=diff")
 	mode  = flag.String("mode", "", "formatting mode: check, diff, or fix (default fix)")
 	path  = flag.String("path", "", "assume BUILD file has this path relative to the workspace directory")
+	tbls  = flag.String("tables", "", "path to JSON file with custom table definitions")
```

Comment:
I concede `tbls` is a terrible name, but I put this up initially for two reasons:

1. Because I'm importing `github.com/bazelbuild/buildifier/tables` above, I can't name it `tables`, can I?
2. If I name it `tablesFlag` or something like that, then am I supposed to update all of the lines above so the `=` signs line up?